app/directives/articles/cbd-article.js

```diff
@@ -15,7 +15,8 @@ import('css!https://cdn.cbd.int/@scbd/ckeditor5-build-inline-full@22.0.0/build/c
 			replace: true,
 			scope: {
 				query : '=query',
-				onLoad: '&onLoad'
+				onLoad: '&onLoad',
+				article: '=?'
```

Comment:
Would it be preferable to use one-way binding
```suggestion
				article: '<?'
```